src/components/Panels/PropertiesPanel/PropertiesPanel.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+    <div
+        id="moduleProperty"
+        class="moduleProperty noSelect effect1 properties-panel draggable-panel draggable-panel-css guide_2"
+    >
+        <PanelHeader :header-title="$t('simulator.panel_header.properties')" />
+        <div class="panel-body">
+            <div id="moduleProperty-inner">
+                <div id="moduleProperty-header">{{ panelBodyHeader }}</div>
+                <PanelType1 v-if="panelType == 1" />
+                <PanelType2 v-if="panelType == 2" />
```

Comment:
If at a time, only one of these PanelType is to be rendered, then it is best practice to chain v-if with v-else-if.

Reply:
Okay will change it